netbout-web/pom.xml

```diff
@@ -736,6 +736,11 @@
                                 <exclude>checkstyle:/src/main/scss/.*</exclude>
                                 <exclude>checkstyle:/src/test/casperjs/.*</exclude>
                                 <exclude>checkstyle:/src/main/js/.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in web
+                                 filters in qulice 0.12 are broken, right excludes
+                                 should be more granular
+                                -->
                                 <exclude>findbugs:.*</exclude>
```

Comment:
@dmzaytsev I created a ticket but suggested to try newer qulice version. In newer version(0.15.2) it is fixed. The problem is that Netbout uses 0.12